# Fix error in `pyproject.toml` on Python 3.13+ (strict PEP 621).
# Updates `license` from string to table format.
# Author: Emik Lin (HKUMed CPOS)
--- pyproject.toml.orig	2025-10-21 04:36:33.000000000 +0800
+++ pyproject.toml	2026-01-23 09:41:59.287873026 +0800
@@ -8,7 +8,7 @@
 description = "Setuptools extension for CalVer package versions"
 readme = "README.md"
 requires-python = ">=3.9"
-license = "Apache-2.0"
+license = {text = "Apache-2.0"}
 authors = [
     { name = "Dustin Ingram", email = "di@python.org" }
 ]
